proxy/delete.go

```diff
@@ -48,7 +48,7 @@ func (c *Client) DeleteFunction(ctx context.Context, functionName string, namesp
 	case http.StatusOK, http.StatusCreated, http.StatusAccepted:
 		fmt.Println("Removing old function.")
 	case http.StatusNotFound:
-		fmt.Println("No existing function to remove")
+		err = fmt.Errorf("No existing function to remove")
 	case http.StatusUnauthorized:
 		fmt.Println("unauthorized access, run \"faas-cli login\" to setup authentication for this server")
```

Comment:
Shouldn't we also returns an error from proxy function when it returns Unauthorized ?

Reply:
Yes, is should, I just updated it.